invenio/modules/knowledge/admin.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2014, 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Knowledge Flask Blueprint."""
+
+import os
+
+from flask import request
+
+from invenio.base.i18n import _
+from invenio.ext.admin.views import ModelView
+from invenio.ext.sqlalchemy import db
+from invenio.modules.search.models import Collection
+
+from .forms import KnowledgeForm, DynamicKnowledgeForm, WrittenAsKnowledgeForm, \
+    TaxonomyKnowledgeForm, KnwKBRVALForm
+from .models import KnwKB, KnwKBRVAL
+
+
+def KnwKBRVAL_id_knwKB_formatter(view, context, model, name):
+    """Formatter for the id_knwKB: return the Knowledge name instead."""
+    return model.kb.name
+
+
+def Knowledge_kbtype_formatter(view, context, model, name):
+    """Formatter for the kbtype: return the Knowledge name instead."""
+    value = model.kbtype
+    return next((key for key, val in KnwKB.KNWKB_TYPES.items() if val == value),
+                None)
+
+
+class KnwKBRVALAdmin(ModelView):
+
+    """Flask-Admin module to manage Knowledge Base Mappings."""
+
+    # form_columns = ('m_key', 'm_value', 'kb')
+    form = KnwKBRVALForm
+
+    column_list = ['id_knwKB', 'm_key', 'm_value']
+    column_labels = dict(
+        id_knwKB='Knowledge',
+        m_key='Map From',
+        m_value='To')
```

Comment:
What about following style?
```python
    m_value='To',
)
```
This is matter of preference. We don't have any strict recommendation for it.